bitsv/network/services/bitindex3.py

```diff
@@ -61,71 +65,291 @@ def get_balance(self, address):
+
+        :param xpub: Xpub to query utxos
+        """
+        r = requests.get(
+            f'https://api.bitindex.network/api/v3/{self.network}/xpub/{xpub}/status',
+            headers=self.authorized_headers,
+        )
+        return r.json()
+
+    def get_xpub_utxos(self, xpub, sort=None):
+        """
+        Get utxos for xpub
+
+        :param xpub: Xpub to query utxos
+        :param sort: Format is 'field:asc' such as 'value:desc' to sort by value descending
+        """
+        r = requests.get(
+            f'https://api.bitindex.network/api/v3/{self.network}/xpub/{xpub}/utxo',
+            params={'sort': sort},
+            headers=self.authorized_headers,
+        )
+        return r.json()
+
+    def get_xpub_transactions(self, xpub):
+        """
+        Get transaction history for xpub
+
+        :param xpub: Xpub to query utxos
         """
         r = requests.get(
-            f'https://api.bitindex.network/api/v3/{self.network}/status?q={q}',
+            f'https://api.bitindex.network/api/v3/{self.network}/xpub/{xpub}/txs',
+            headers=self.authorized_headers,
+        )
+        return r.json()
+
+    def get_webhook_config(self):
+        """
+        Get the configured webhook endpoint
+        """
+        r = requests.get(
+            f'https://api.bitindex.network/api/v3/{self.network}/webhook/endpoint',
+            headers=self.authorized_headers,
+        )
+        return r.json()
+
+    def update_webhook_config(self, url, enabled, secret):
+        """
+        Update webhook endpoint confirmation
+
+        :param url: Url to callback when monitored address or xpub has a payment
+        :param enabled: Whether webhooks are enabled or disabled
+        :param secret: Secret parameter passed back to your API for security purposes
+        """
+        r = requests.put(
+            f'https://api.bitindex.network/api/v3/{self.network}/webhook/endpoint',
+            data=json.dumps({
+                'url': url,
+                'enabled': enabled,
+                'secret': secret,
+            }),
+            headers=self.authorized_headers,
+        )
+        return r.json()
+
+    def get_webhook_monitored_addresses(self):
+        """
+        Get monitored addresses and xpubs
+        """
+        r = requests.get(
+            f'https://api.bitindex.network/api/v3/{self.network}/webhook/monitored_addrs',
+            headers=self.authorized_headers,
+        )
+        return r.json()
+
+    def update_webhook_monitored_addresses(self, address):
+        """
+        Update monitored addresses and xpubs
+
+        :param address: Address or xpub key to track and monitor
+        """
+        r = requests.put(
+            f'https://api.bitindex.network/api/v3/{self.network}/webhook/monitored_addrs',
+            data=json.dumps({'addr': address}),
+            headers=self.authorized_headers,
+        )
+        return r.json()
+
+    def webhook_sample_callback(
```

Comment:
@AustEcon This is not a 'real' API call.

I should have been more explicit in the docs.  This is a SAMPLE of what the user will receive once they have register an endpoint to callback to their URL.

Reply:
@attilaaf thanks for the clarification. This is removed in https://github.com/AustEcon/bitsv/pull/23